<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantPageBugAwareness</class>
 <widget class="QWidget" name="AssistantPageBugAwareness">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>484</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_detailLabel">
     <property name="text">
      <string comment="@info refers to 'if you can give detailed information'">&lt;para&gt;&lt;b&gt;Can you give detailed information about what you were doing when the application crashed?&lt;/b&gt;&lt;/para&gt;&lt;para&gt;If you can, that will help the developers reproduce and fix the bug. &lt;emphasis&gt;Examples of Details: steps to take to reproduce the crash, what you were doing in the application, whether it happens every time, or whether it crashes on a particular URL or document.&lt;/emphasis&gt;&lt;/para&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_canDetailCheckBox">
     <property name="text">
      <string comment="@option:check">Yes, I can give information about the crash.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="m_developersCanContactReporterLabel">
     <property name="text">
      <string comment="@info">&lt;para&gt;&lt;b&gt;Can the developers contact you for more information?&lt;/b&gt;&lt;/para&gt;&lt;para&gt;Sometimes, while the bug is being investigated, the developers need more information from the reporter in order to fix the bug.&lt;/para&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_developersCanContactReporterCheckBox">
     <property name="text">
      <string comment="@option:check">Yes, developers can contact me for more information if required.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
